junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/execution/NamespaceAwareStore.java

```diff
@@ -27,12 +26,12 @@
  * @since 5.0
  */
 @API(status = INTERNAL, since = "5.0")
-public class NamespaceAwareStore implements Store {
+public class NamespaceAwareStore<N> implements Store {
```

Comment:
I think the type parameter I introduced here can be removed again. We only ever use the `Namespace` from junit-platform-engine in production code.

Reply:
When a type parameter is only used by tests, we should rather refactor the tests. In this case, they should be able to work with the `Namespace` from junit-platform-engine.